observability/monitoring/common-monitoring-configuration-scenarios.adoc

```diff
@@ -0,0 +1,57 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="common-monitoring-configuration-scenarios"]
+= Common monitoring configuration scenarios
+include::_attributes/common-attributes.adoc[]
+:context: common-monitoring-configuration-scenarios
+
+toc::[]
+
+After {product-title} is installed, core platform monitoring components immediately begin collecting metrics, which you can query and view.
+The default in-cluster monitoring stack includes the core platform Prometheus instance that collects metrics from your cluster and the core Alertmanager instance that routes alerts, among other components.
+Depending on who will use the monitoring stack and for what purposes, as a cluster administrator, you can further configure these monitoring components to suit the needs of different users in various scenarios.
+
+In addition to core platform monitoring, you can also optionally xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[enable monitoring for user-defined projects] for user workload monitoring.
+Users can then monitor their own services and workloads without the need for an additional monitoring solution.
+
+[id="configuring-core-platform-monitoring-first-steps_{context}"]
+== Configuring core platform monitoring: First steps
```

Comment:
Just a suggestion: the wording of this title made me think that there might be "second steps" or "next steps". It might be helpful to change it to avoid this confusion. Maybe something like, "Configuring core platform monitoring: post-installation steps" or something similar.

Reply:
The plan is to add a few more "advanced" scenarios eventually as part of ContentX work, but I think you make a valid point. I'll adjust the wording for now until we have this other content added. Thanks!